mithril-aggregator/src/runtime/state_machine.rs

```diff
@@ -281,22 +307,26 @@ impl AggregatorRuntime {
                 nested_error: None,
             })?;
 
-        // Every error raised below this comment must be raised to critical in order to trigger an aggregator restart which will avoid a dead loop in the state machine
-        // TODO: Implement a graceful retry mechanism for artifact creation and label open messages with `is_artifact_created` (to keep track of certified but without artifacts open messages)
-        self.runner.drop_pending_certificate().await.map_err(|e| {
-            RuntimeError::critical(
-                "transiting SIGNING → READY: failed dropping pending certificate",
-                Some(e),
-            )
-        })?;
+        // Every error raised below this comment must be raised to critical in
+        // order to trigger an aggregator restart which will avoid a dead loop
+        // in the state machine TODO: Implement a graceful retry mechanism for
```

Comment:
The `todo` should start on a new line

Reply:
The `TODO` part can be removed as a retry mechanism is now implemented in the signed entity service